suites/squid/common/sanity/01-nvmeof.yaml

```diff
@@ -0,0 +1,33 @@
+tests:
```

Comment:
Please include NVMe HA sanity tests, from this suite file, [tier-1_nvmeof_ha_sanity.yaml](https://github.com/red-hat-storage/cephci/blob/master/suites/squid/nvmeof/tier-1_nvmeof_ha_sanity.yaml).
Let me know if you need any help.

Reply:
@sunilkumarn417 will these tests work on the config https://github.com/red-hat-storage/cephci/blob/5363573baf304678c491f9226507cc2761259785/conf/squid/common/1admin-7node-2client.yaml ?
If no, please point out necessary changes.

Reply:
Please use this test cases against `1admin-7node-2client.yaml` configuration files.
```
  #  Configure Ceph NVMeoF gateway
  #  Configure Initiators
  #  Run IO on NVMe Targets
  - test:
      abort-on-fail: true
      config:
        rbd_pool: rbd
        gw_group: gw_group1
        do_not_create_image: true
        rep-pool-only: true
        rep_pool_config:
          pool: rbd
        install: true                           # Run SPDK with all pre-requisites
        cleanup:
          - pool
          - gateway
          - initiators
        gw_nodes:
          - node6
          - node7
        subsystems:                             # Configure subsystems with all sub-entities
          - nqn: nqn.2016-06.io.spdk:cnode1
            serial: 1
            bdevs:
            - count: 2
              size: 2G
              lb_group: node6
            listener_port: 4420
            listeners:
              - node6
              - node7
            allow_host: "*"
          - nqn: nqn.2016-06.io.spdk:cnode2
            serial: 2
            bdevs:
            - count: 2
              size: 2G
              lb_group: node7
            listener_port: 4420
            listeners:
              - node6
              - node7
            allow_host: "*"
        initiators:                             # Configure Initiators with all pre-req
          - nqn: connect-all
            listener_port: 4420
            node: node8
        fault-injection-methods:                # Failure induction
          - tool: systemctl
            nodes: node6
      desc: NVMEoF failover-failback test via systemctl rpm tool
      destroy-cluster: false
      module: test_ceph_nvmeof_high_availability.py
      name: NVMeoF 2-GW HA failover-failback via systemctl
      polarion-id: CEPH-83588636

  - test:
      abort-on-fail: true
      config:
        rbd_pool: rbd
        gw_group: gw_group1
        do_not_create_image: true
        rep-pool-only: true
        rep_pool_config:
          pool: rbd
        install: true                           # Run SPDK with all pre-requisites
        cleanup:
          - pool
          - gateway
          - initiators
        gw_nodes:
          - node6
          - node7
        subsystems:                             # Configure subsystems with all sub-entities
          - nqn: nqn.2016-06.io.spdk:cnode1
            serial: 1
            bdevs:
            - count: 2
              size: 2G
              lb_group: node6
            listener_port: 4420
            listeners:
              - node6
              - node7
            allow_host: "*"
          - nqn: nqn.2016-06.io.spdk:cnode2
            serial: 2
            bdevs:
            - count: 2
              size: 2G
              lb_group: node7
            listener_port: 4420
            listeners:
              - node6
              - node7
            allow_host: "*"
        initiators:                             # Configure Initiators with all pre-req
          - nqn: connect-all
            listener_port: 4420
            node: node8
        fault-injection-methods:                # Failure induction
          - tool: daemon
            nodes: node6
      desc: NVMEoF failover-failback test via cephadm daemon orchestration
      destroy-cluster: false
      module: test_ceph_nvmeof_high_availability.py
      name: NVMeoF 2-GW HA failover-failback via orchestrator
      polarion-id: CEPH-83588699
```